### DISCUSSION TITLE
All my argoCD projects are gone(?)

### DISCUSSION THREAD

**Discussion by rhorv** [argoproj/argo-cd#6802] [Q&A]
Hi,

I am new to argocd, so I was hoping someone can help me solve this problem.

Background: 
I wanted to restart argoCD in insecure mode, so I went in and added the "--insecure" flag to my existing argocd deployment, and apply the config that I had saved from originally deploying the service. I quickly noticed that I forgot to specify the custom arogcd namespace, so it dumped everything as new into "default". I don't use the default namespace so I just went ahead and deleted it using the same config, not specifying the namespace again. 

After that I apply the config with the right namespace this time and it noted the change in the deployment. 
However after logging in I discovered that all my projects and applications are gone. The services are still deployed and running fine, but I lost everything within argoCD, except for the repository definitions, pretty much.

Is there any way to do some sort of auto-discovery on the cluster for projects? Why are they even gone? Was that the delete from another namespace that somehow deleted everything even though it wasn't even supposed to touch the "other" installation?
Is there any way to recover this?

Any advise is greatly appreciated.

Thank you

**Comment by meier-christoph** [comment]
Hi, you probably deleted the CRD which is cluster scoped and if you delete a CRD it also deletes all resources that go with it.

https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/#delete-a-customresourcedefinition

Sorry but they are gone for good.
